ledger-core/insolar/payload/validation.go

```diff
@@ -0,0 +1,178 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package payload
+
+import (
+	"github.com/insolar/assured-ledger/ledger-core/vanilla/throw"
+)
+
+type Validate interface {
```

Comment:
I would rather move validation to it's own subpackage. From my point on view payload package should be only about marshalling/unmarshalling.